integration/container/kill_test.go

```diff
@@ -175,3 +175,21 @@ func TestKillStoppedContainerAPIPre120(t *testing.T) {
 	err = client.ContainerKill(ctx, c.ID, "SIGKILL")
 	require.NoError(t, err)
 }
+
+func TestKillDifferentUserContainer(t *testing.T) {
+	// TODO Windows: Windows does not yet support -u (Feb 2016).
+	skip.If(t, testEnv.OSType != "linux")
```

Comment:
Should we print "Windows does not support -u" in the skip ?

Reply:
Well the message should be different as we're not using the `cli` at all

Reply:
@vdemeester Updated. Thanks.